selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/AbstractDevice.java

```diff
@@ -567,10 +566,10 @@ public String listRunningThirdPartyProcesses() {
     String psOutput = runAdbCommand("shell ps");
     StringBuilder sb = new StringBuilder();
     boolean isFirstHeaderLine = true;
-    for (String line: Splitter.on("\n").split(psOutput)) {
+    for (String line: Splitter.on(System.getProperty("line.separator")).split(psOutput)) {
```

Comment:
Well that's embarrassing. No it doesn't. I guess the '\n' problems from SelendroidResponseTest led me to believe they were causing problems here too.